test/e2e/tests/metrics.spec.js

```diff
@@ -30,6 +30,7 @@ describe('Segment metrics', function () {
         fixtures: 'metrics-enabled',
         ganacheOptions,
         title: this.test.title,
+        failOnConsoleError: false,
```

Comment:
Sentry requests are mocked in all tests except this test since we [reset](https://github.com/MetaMask/metamask-extension/blob/sentry-e2e/test/e2e/tests/metrics.spec.js#L6) the mock server

The failOnConsoleError arg is needed else the test will now fail on a console error, caused by a request to sentry thats not intercepted and uses an invalid client key `fake` and project id `0000000`
```
https://sentry.io/api/0000000/envelope/?sentry_key=fake&sentry_version=7 - Failed to load resource: the server responded with a status of 400 (Bad Request)
```

Reply:
Hmm, could we solve this by creating a second mock server?

Not a blocker, and I don't know how to best do that without overly complicating the test. Just it's nice to leave `failOnConsoleError` enabled, that has caught issues before.

Reply:
I'll think of what options we have, and can update the test in a new branch shortly.